EventFinder-DDLs.sql

```diff
@@ -0,0 +1,125 @@
+CREATE TABLE DoB (
+    DateOfBirth DATE PRIMARY KEY,
+    Age INT NOT NULL
+);
+
+CREATE TABLE User (
+    UserID INT PRIMARY KEY AUTO_INCREMENT,
+    FirstName VARCHAR(50) NOT NULL,
+    LastName VARCHAR(50) NOT NULL,
+    DateOfBirth DATE NOT NULL,
+    Gender VARCHAR(50) NOT NULL,
+    FOREIGN KEY (DateOfBirth) REFERENCES DoB(DateOfBirth) ON DELETE CASCADE ON UPDATE CASCADE
+);
+
+CREATE TABLE CreditCard (
+    CardNumber VARCHAR(16) PRIMARY KEY,
+    ExpiryDate DATE NOT NULL,
+    HolderName VARCHAR(100) NOT NULL,
+    CVC INT NOT NULL
+);
+
+CREATE TABLE Organizer (
+    OrganizerUserID INT PRIMARY KEY AUTO_INCREMENT,

```

Comment:
This is also a foreign key referring to the user table. Therefore we don't need `AUTO_INCREMENT` here.